src/topology/algebra/monoid.lean

```diff
@@ -272,6 +273,37 @@ lemma continuous.pow {f : α → M} [topological_space α] (h : continuous f) (n
   continuous (λ b, (f b) ^ n) :=
 continuous.comp (continuous_pow n) h
 
+namespace homeomorph
+
+variables [topological_space α] [group_with_zero α] [has_continuous_mul α]
+
```

Comment:
@sgouezel We have `topology/algebra/group_with_zero`, should these lemmas live there rather than in `topology_algebra/monoid`?

Reply:
Yes, I agree. Moved.